contracts/src/Types.sol

```diff
@@ -14,6 +14,22 @@ function ParaIDNe(ParaID a, ParaID b) pure returns (bool) {
     return !ParaIDEq(a, b);
 }
 
+function into(ParaID paraID) pure returns (ChannelID) {
```

Comment:
This converts a para_id into a channelID. Same as `impl From<ParaId> for ChannelId` on the substrate side.